app/src/main/java/com/eventyay/organizer/data/db/configuration/OrgaDatabase.java

```diff
@@ -310,4 +311,20 @@ public void onPreMigrate() {
         }
     }
 
+    @Migration(version = 20, database = OrgaDatabase.class)
```

Comment:
`version = VERSION`

Reply:
Wouldn't it be better to keep it like this only since when migrating to the next version we'll have to change this again? Because older versions are not removed.

Reply:
`version = VERSION` is wrong. Migrations will never run correctly if written that way